docs/content/en/blog/news/control-plane-on-ecs.md

```diff
@@ -0,0 +1,283 @@
+    "valueFrom" : "arn:aws:secretsmanager:${data.aws_region.current.id}:${data.aws_caller_identity.self.account_id}:secret:${var.control_plane_config_secret}"
+  },
+]
+},
+```
+
+2. ops
+```
+{
+name  = "pipecd-ops"
+image = var.ops_image_url
+portMappings = [
+  {
+    "name" : "http",
+    "protocol" : "tcp",
+    "containerPort" : 9082,
+    "appProtocol" : "http"
+  },
+  {
+    "name" : "http",
+    "protocol" : "tcp",
+    "containerPort" : 9085,
+    "appProtocol" : "http"
+  },
+]
+command = [
+  "/bin/sh -c 'echo $CONTROL_PLANE_CONFIG; echo $CONTROL_PLANE_CONFIG | base64 -d >> control-plane-config.yaml; sed -i -e s/pipecd-mysql/${var.db_instance_address}/ control-plane-config.yaml; echo $ENCRYPTION_KEY >> encryption-key; pipecd ops --cache-address=${var.redis_host}:6379 --config-file=control-plane-config.yaml --log-encoding=humanize --metrics=true;'"
+]
+entrypoint = [
+  "sh",
+  "-c"
+]
+secrets = [
+  {
+    "name" : "ENCRYPTION_KEY",
+    "valueFrom" : "arn:aws:secretsmanager:${data.aws_region.current.id}:${data.aws_caller_identity.self.account_id}:secret:${var.encryption_key_secret}"
+  },
+  {
+    "name" : "CONTROL_PLANE_CONFIG",
+    "valueFrom" : "arn:aws:secretsmanager:${data.aws_region.current.id}:${data.aws_caller_identity.self.account_id}:secret:${var.control_plane_config_secret}"
+  },
+]
+},
+```
+
+### ALB
+You must prepare two target group for both of HTTP and gRPC. Make two hosts and listner rules as below. Listner protocol should be HTTPS becuase it use gRPC.
+
+![](/images/control-plane-alb.png)
+
+### Terraform example
+PipeCD gives [control-plane-aws-ecs-terraform-demo](https://github.com/pipe-cd/control-plane-aws-ecs-terraform-demo)
+
+#### Prepare
+1. Prepare SSL certificate
+Prepare your domain and SSL certificate from AWS certificate manager.
+
+2. Create a s3 bucket for terraform backend
+Write bucket name to `00-main.tf`
+```
+terraform {
+  backend "s3" {
+    bucket  = "example-pipecd-control-plane-tfstate" #your bucket name for terraform backend
+    region  = "ap-northeast-1"
+    key     = "tfstate"
+    profile = "pipecd-control-planeg-terraform" #your profile
+  }
+  required_providers {
+    aws = {
+      version = "~> 3.34.0"
+    }
+  }
+}
+```
+
+3. Edit `variables.tf` for your project
+```
+//export
+locals {
+  alb = {
+    certificate_arn = ""
+  }
+  
+  redis = {
+    node_type = "cache.t2.micro"
+  }
+
+  rds = {
+    node_type = "db.t3.micro"
+  }
+
+  ecs = {
+    memory = "1024"
+    cpu = "512"
+  }
+}
+```
+
+4. Create a s3 bucket for filestore and write the bucket name for it to `control-plane-config.yaml` and `variables.tf`
+```
+apiVersion: "pipecd.dev/v1beta1"
+kind: ControlPlane
+spec:
+    datastore:
+    type: MYSQL
+    config:
+        url: root:test@tcp(pipecd-mysql:3306)
+        database: quickstart
+    filestore:
+    type: S3
+    config: # edit here
+        bucket: example-pipecd-control-plane-filestore 
+        region: ap-northeast-1
```

Comment:
Yaml indent is wrong. `datastore` and `filestore` attributes should have `type` and `config` as child.